CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to `c8`
+
+The `c8` project welcomes all contributions from anyone willing to work in good faith with other contributors and the community. No contribution is too small and all contributions are valued.
+
+## Issues
+
+- You can open [issues here](https://github.com/bcoe/c8/issues), please follow the template guide.
+- You can come over to the `node-tooling/c8` channel for discussions, [follow this link](https://devtoolscommunity.herokuapp.com/) to request for an invite.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code, documentation, dependencies, and tools contained in the `c8` repository.
+
+### Setting up your local environment
+
+- Make sure you have installed the latest version of Node.js
```

Comment:
just a minor thing, you could just use `1.` for every bullet point and markdown will figure out the numbering, even if we have to add or remove steps later.